examples/inspection/plot_permutation_importance.py

```diff
@@ -20,8 +20,8 @@
 
 .. topic:: References:
 
-   [1] L. Breiman, "Random Forests", Machine Learning, 45(1), 5-32,
-       2001. https://doi.org/10.1023/A:1010933404324
+   [1] :doi:`L. Breiman, "Random Forests", Machine Learning, 45(1), 5-32,
+       2001. <10.1023/A:1010933404324>`
```

Comment:
The [rendering](https://178355-843222-gh.circle-artifacts.com/0/doc/auto_examples/inspection/plot_permutation_importance.html) isn't correct: it is not in the [current version](https://scikit-learn.org/dev/auto_examples/inspection/plot_permutation_importance.html) too.
```suggestion
   :doi:`L. Breiman, "Random Forests", Machine Learning, 45(1), 5-32,
   2001. <10.1023/A:1010933404324>`
```
I think this could fix the rendering